diff --git a/tmp/tmp.IDokXWtuJT b/tmp/tmp.3OuSqciTlL
index 4735904..d86ab8d 100644
--- a/tmp/tmp.IDokXWtuJT
+++ b/tmp/tmp.3OuSqciTlL
@@ -1,4 +1,4 @@
-数値流体力学第6回講義 非構造格子系の手法 (**有限体積法・有限要素法など**)
+数値流体力学第6回講義非構造格子系の手法 (**有限体積法・有限要素法など**)
 による流体解析 (2)
 三目直登
 
@@ -18,7 +18,7 @@
 Y. Bazilevs, K. Takizawa and T.E. Tezduyar, Computational Fluid– Structure Interaction, Wiley, 2013.
 
 Y. Bazilevs, K. Takizawa and T.E. Tezduyar,
-(訳: 津川 祐美子, 滝沢研二), 流体-**構造連成** 問題の数値解析, 森北出版\*\*, 2015.\*\*
+(訳: 津川祐美子, 滝沢研二), 流体-**構造連成** 問題の数値解析, 森北出版\*\*, 2015.\*\*
 
 
 日本計算工学会流れの有限要素法研究委員会, 続・有限要素法による流れのシミュレーション, シュプリンガージャパン\*\*, 2008.\*\*
@@ -67,7 +67,7 @@ Y. Bazilevs, K. Takizawa and T.E. Tezduyar,
 
 * **格子間のフラックス** (流束)
 * **保存性を担保する工夫**
-  ガウスの発散定理を用いて非構造格子間のフラック を表していると解釈するこ ス収支を求める問題に変更 し、解析する手法
+  ガウスの発散定理を用いて非構造格子間のフラックを表していると解釈するこス収支を求める問題に変更し、解析する手法
 * 非構造格子 (コントロールボリューム) **が使用可能**
 
 
@@ -128,7 +128,7 @@ s=1 Si,sFˆ (φi, φconn(i,s)) · ndΓ = ViS(φi Vi ) ∂φi
 
 6 s=1 Si,sFˆ (φi, φconn(i,s)) · ndΓ = ViS(φi + ) !
 
-6 s=1 Si,sFˆ (Fi,Fconn(i,s)) · n j = conn(i, s): i 番のCV内にある表面 s の向こう側のCV番号を返す関数 (**連結関係リスト**) j = conn(i, s)
+6 s=1 Si,sFˆ (Fi,Fconn(i,s)) · n j = conn(i, s): i番のCV内にある表面sの向こう側のCV番号を返す関数 (**連結関係リスト**) j = conn(i, s)
 
 # 一般化保存則の離散化
 
@@ -138,7 +138,7 @@ s=1 Si,sFˆ (φi, φconn(i,s)) · ndΓ = ViS(φi Vi ) ∂φi
 \#dΩ =
 !
 
-各表面の積分 の和に変形
+各表面の積分の和に変形
 ∂t dΩ +"
 6 Ωi
 ∂φ Ωi S(φ)dΩ
@@ -195,10 +195,10 @@ $$\frac{\partial\varphi}{\partial t}+\nabla\cdot\mathbf{F}(\phi)=S(\phi)$$
 ## →差分法など
 
 * 計算点の連結情報が自明であり、
-  行列の形も特徴的 (三重対角行列 など) なので、計算コストが低い
-* 座標変換による境界適合格子などの工夫が可能だが、空 間の表現に限界がある 非構造格子: 四面体など任意形状のメッシュを使用
+  行列の形も特徴的 (三重対角行列など) なので、計算コストが低い
+* 座標変換による境界適合格子などの工夫が可能だが、空間の表現に限界がある非構造格子: 四面体など任意形状のメッシュを使用
   →有限体積法・有限要素法など
-* 計算点の連結情報が必要であり、計算コスト が高い
+* 計算点の連結情報が必要であり、計算コストが高い
 * 複雑形状の対象にも利用できる
 
 ## 有限体積法
@@ -212,7 +212,7 @@ $$\frac{\partial\varphi}{\partial t}+\nabla\cdot\mathbf{F}(\phi)=S(\phi)$$
 
 ## メッシュフリー法
 
-→Smoothed Particle Hydrodynamics 法・Element-Free Galerkin 法など
+→Smoothed Particle Hydrodynamics法・Element-Free Galerkin法など
 
 * メッシュ作成のコストやメッシュの歪みによる計算不安定性が少ない
 * 表面などの幾何学的情報が一部失われるので、境界等の精度が低い
@@ -221,15 +221,15 @@ $$\frac{\partial\varphi}{\partial t}+\nabla\cdot\mathbf{F}(\phi)=S(\phi)$$
 
 1. **連続体力学と変分法の関係** 2. **有限要素法の基本**
    2.1. **重み付き残差法と弱形式化**
-   2.2. Galerkin 法 2.3. 形状関数による補間 3. **安定化有限要素法**
-   3.1. Upwind Galerkin 法 3.2. Streamline upwind Petrov/Gakerkin 法 3.3. **その他の安定化スキーム**
+   2.2. Galerkin法2.3. 形状関数による補間3. **安定化有限要素法**
+   3.1. Upwind Galerkin法3.2. Streamline upwind Petrov/Gakerkin法3.3. **その他の安定化スキーム**
 
 # 目次
 
 1. **連続体力学と変分法の関係** 2. **有限要素法の基本**
    2.1. **重み付き残差法と弱形式化**
-   2.2. Galerkin 法 2.3. 形状関数による補間 3. **安定化有限要素法**
-   3.1. Upwind Galerkin 法 3.2. Streamline upwind Petrov/Gakerkin 法 3.3. **その他の安定化スキーム**
+   2.2. Galerkin法2.3. 形状関数による補間3. **安定化有限要素法**
+   3.1. Upwind Galerkin法3.2. Streamline upwind Petrov/Gakerkin法3.3. **その他の安定化スキーム**
 
 
 
@@ -244,15 +244,15 @@ $$\frac{\partial\varphi}{\partial t}+\nabla\cdot\mathbf{F}(\phi)=S(\phi)$$
 
 
 
-無限個の質点が伱間なく並んで 連続的な分布を形成するよう拡張 連続体力学 レオロジー
+無限個の質点が伱間なく並んで連続的な分布を形成するよう拡張連続体力学レオロジー
 
 
 固体
 (弾性体)
-弾性体力学 構造力学 材料力学 流体力学 水理学 流体 = 静止状態においてせん断応力が発生しない連続体
-※せん断応力のイメージは、物体表面の接線方向に働く力 連続体力学学習におすすめの書籍: よくわかる連続体力学ノート, 京谷孝史, 森北出版.
+弾性体力学構造力学材料力学流体力学水理学流体 = 静止状態においてせん断応力が発生しない連続体
+※せん断応力のイメージは、物体表面の接線方向に働く力連続体力学学習におすすめの書籍: よくわかる連続体力学ノート, 京谷孝史, 森北出版.
 
-11 流体
+11流体
 
 # 1. 連続体力学と変分法の関係
 
@@ -270,98 +270,98 @@ $$\frac{\partial\varphi}{\partial t}+\nabla\cdot\mathbf{F}(\phi)=S(\phi)$$
 変分法とは?
 
 高校数学→関数の最大値・最小値問題 (有限次元の微分)
-関数: 数値 → 数値 解法: を満たす x を求める 変分法 →汎関数の最大値・最小値問題 (無限次元の微分: 汎関数微分)
+関数: 数値 → 数値解法: を満たすxを求める変分法 →汎関数の最大値・最小値問題 (無限次元の微分: 汎関数微分)
 
 # 1. **連続体力学と変分法の関係**
 
 変分法とは?
 
 高校数学→関数の最大値・最小値問題 (有限次元の微分)
-関数: 数値 → 数値 解法: を満たす x を求める 変分法 →汎関数の最大値・最小値問題
+関数: 数値 → 数値解法: を満たすxを求める変分法 →汎関数の最大値・最小値問題
 →**変分問題**
 (無限次元の微分: 汎関数微分)
 y!=
-dy 汎関数: 関数 → 数値 dx
+dy汎関数: 関数 → 数値dx
 
 # 1. **連続体力学と変分法の関係**
 
 変分法とは?
 
 高校数学→関数の最大値・最小値問題 (有限次元の微分)
-関数: 数値 → 数値 解法: を満たす x を求める 変分法 →汎関数の最大値・最小値問題
+関数: 数値 → 数値解法: を満たすxを求める変分法 →汎関数の最大値・最小値問題
 →**変分問題**
 (無限次元の微分: 汎関数微分)
 y!=
-dy 汎関数: 関数 → 数値 dx 解法: を満たす (停留関数) を求める
+dy汎関数: 関数 → 数値dx解法: を満たす (停留関数) を求める
 →汎関数微分 (変分) **による解法**
-Euler–Lagrange 方程式
+Euler–Lagrange方程式
 
 # 1. **連続体力学と変分法の関係**
 
 変分法とは?
 
 高校数学→関数の最大値・最小値問題 (有限次元の微分)
-関数: 数値 → 数値 解法: を満たす x を求める 変分法 →汎関数の最大値・最小値問題
+関数: 数値 → 数値解法: を満たすxを求める変分法 →汎関数の最大値・最小値問題
 →**変分問題**
 (無限次元の微分: 汎関数微分)
 y!=
-dy 汎関数: 関数 → 数値 dx 解法: を満たす (停留関数) を求める
+dy汎関数: 関数 → 数値dx解法: を満たす (停留関数) を求める
 →汎関数微分 (変分) **による解法**
-Euler–Lagrange 方程式 変分: 関数 を微小に変化させて としたときの汎 y(x) y(x) + δy(x)
-関数の微小変化の値で、2次の微小量を無視したもの δI ! I\[y + δy] − I\[y]
+Euler–Lagrange方程式変分: 関数を微小に変化させてとしたときの汎y(x) y(x) + δy(x)
+関数の微小変化の値で、2次の微小量を無視したもの δI ! I\[y + δy]− I\[y]
 
 # 1. **連続体力学と変分法の関係**
 
 変分法とは?
 
 高校数学→関数の最大値・最小値問題 (有限次元の微分)
-関数: 数値 → 数値 解法: を満たす x を求める 変分法 →汎関数の最大値・最小値問題
+関数: 数値 → 数値解法: を満たすxを求める変分法 →汎関数の最大値・最小値問題
 →**変分問題**
 (無限次元の微分: 汎関数微分)
 y!=
-dy 汎関数: 関数 → 数値 dx 解法: を満たす (停留関数) を求める
-→汎関数微分 (変分) による解法 Euler–Lagrange 方程式 最小作用の原理 **(principle of least action):**
-力学系の運動は、作用(解析力学で定義される汎関数)が最小にな るような運動である
+dy汎関数: 関数 → 数値dx解法: を満たす (停留関数) を求める
+→汎関数微分 (変分) による解法Euler–Lagrange方程式最小作用の原理 **(principle of least action):**
+力学系の運動は、作用(解析力学で定義される汎関数)が最小になるような運動である
 →力学問題を変分問題として捉えることができる
 
 # 1. **連続体力学と変分法の関係** 変分問題の例
 
-屈折率が連続的に変化する媒質内での光の経路 光の経路: 媒質の屈折率:
-光が経路 C (右図の P0 から P1 まで) を通過するのにか かる時間を求める.
+屈折率が連続的に変化する媒質内での光の経路光の経路: 媒質の屈折率:
+光が経路C (右図のP0からP1まで) を通過するのにかかる時間を求める.
 
 
 # 1. **連続体力学と変分法の関係** 変分問題の例
 
-屈折率が連続的に変化する媒質内での光の経路 光の経路: 媒質の屈折率:
-光が経路 C (右図の P0 から P1 まで) を通過するのにか かる時間を求める.
+屈折率が連続的に変化する媒質内での光の経路光の経路: 媒質の屈折率:
+光が経路C (右図のP0からP1まで) を通過するのにかかる時間を求める.
 
-c を真空中の光の速度とすると、点 (x, y) における光の速度は で与えられ,これを用いると,C 上の微小線素 ds を光が 通過する時間は以下のように表される.
+cを真空中の光の速度とすると、点 (x, y) における光の速度はで与えられ,これを用いると,C上の微小線素dsを光が通過する時間は以下のように表される.
 
 
 # 1. **連続体力学と変分法の関係** 変分問題の例
 
-屈折率が連続的に変化する媒質内での光の経路 光の経路: 媒質の屈折率:
-光が経路 C (右図の P0 から P1 まで) を通過するのにか かる時間を求める.
+屈折率が連続的に変化する媒質内での光の経路光の経路: 媒質の屈折率:
+光が経路C (右図のP0からP1まで) を通過するのにかかる時間を求める.
 
-c を真空中の光の速度とすると、点 (x, y) における光の速度は で与えられ,これを用いると,C 上の微小線素 ds を光が 通過する時間は以下のように表される.
+cを真空中の光の速度とすると、点 (x, y) における光の速度はで与えられ,これを用いると,C上の微小線素dsを光が通過する時間は以下のように表される.
 
-光が経路 C を P0 から P1 まで通過するのにかかる時間は となる.線素 ds の長さ
+光が経路CをP0からP1まで通過するのにかかる時間はとなる.線素dsの長さ
 
 $$T={\frac{1}{c}}\int\_{x\_{0}}^{x\_{1}}n(y(x)){\sqrt{1+y^{\prime}{(x)}^{2}}}d x$$
 
-を用いて,T は以下のように求められる.
+を用いて,Tは以下のように求められる.
 
 
 # 1. **連続体力学と変分法の関係** 変分問題の例
 
-屈折率が連続的に変化する媒質内での光の経路 光の経路: 媒質の屈折率:
-光が経路 C (右図の P0 から P1 まで) を通過するのにか かる時間を求める.
+屈折率が連続的に変化する媒質内での光の経路光の経路: 媒質の屈折率:
+光が経路C (右図のP0からP1まで) を通過するのにかかる時間を求める.
 
-c を真空中の光の速度とすると、点 (x, y) における光の速度は で与えられ,これを用いると,C 上の微小線素 ds を光が 通過する時間は以下のように表される.
+cを真空中の光の速度とすると、点 (x, y) における光の速度はで与えられ,これを用いると,C上の微小線素dsを光が通過する時間は以下のように表される.
 
-光が経路 C を P0 から P1 まで通過するのにかかる時間は となる.線素 ds の長さ を用いて,T は以下のように求められる.
+光が経路CをP0からP1まで通過するのにかかる時間はとなる.線素dsの長さを用いて,Tは以下のように求められる.
 
-光は所要時間が最小になるような経路を通る (フェルマーの原理) より,光の経路は とした変分問題の停留関数として求められる.
+光は所要時間が最小になるような経路を通る (フェルマーの原理) より,光の経路はとした変分問題の停留関数として求められる.
 
 
 # 1. 連続体力学と変分法の関係
@@ -370,7 +370,7 @@ c を真空中の光の速度とすると、点 (x, y) における光の速度
 
 # 1. **連続体力学と変分法の関係**
 
-直交関数展開 関数は無限個の直交関数列として表すことができる 例) **フーリエ級数展開** y(x) = 12 a0 + a1 cos ωx + a2 cos 2ωx + ···
+直交関数展開関数は無限個の直交関数列として表すことができる例) **フーリエ級数展開** y(x) = 12 a0 + a1 cos ωx + a2 cos 2ωx + ···
 
 * b1 sin ωx + b2 sin 2ωx + ···
 
@@ -390,11 +390,11 @@ $$\[|{\mathcal{H}}|\mathbb{H}]$$
 
 # 1. **連続体力学と変分法の関係** 直交関数展開
 
-関数は無限個の直交関数列として表すことができる y(x) = 12 a0 + a1 cos ωx + a2 cos 2ωx + ···
+関数は無限個の直交関数列として表すことができるy(x) = 12 a0 + a1 cos ωx + a2 cos 2ωx + ···
 例) **フーリエ級数展開**
 
 * b1 sin ωx + b2 sin 2ωx + ···
-  関数の直交性 ← Euclid の基底ベクトルの直交性 (内積=0) の拡張
+  関数の直交性 ← Euclidの基底ベクトルの直交性 (内積=0) の拡張
 
 $$e\_{x}\cdot e\_{y}=0$$
 
@@ -405,13 +405,13 @@ $$\overline{{e\_{x}\ \overline{{=}}}}$$
 
 # 1. **連続体力学と変分法の関係** 直交関数展開
 
-関数は無限個の直交関数列として表すことができる y(x) = 12 a0 + a1 cos ωx + a2 cos 2ωx + ···
+関数は無限個の直交関数列として表すことができるy(x) = 12 a0 + a1 cos ωx + a2 cos 2ωx + ···
 例) **フーリエ級数展開**
 
 $\mathbf{a}=\mathbf{a}\cdot\mathbf{a}$.
 
 * b1 sin ωx + b2 sin 2ωx + ···
-  関数の直交性 ← Euclid の基底ベクトルの直交性 (内積=0) の拡張 三角関数の直交性
+  関数の直交性 ← Euclidの基底ベクトルの直交性 (内積=0) の拡張三角関数の直交性
 
 $$(m=n)$$
 $$\overline{{2}}$$
@@ -435,7 +435,7 @@ $$x\prime\cos x$$
 # $\sin(n\omega x)\sin(m\omega x)dx=$  .
 
 
-(Fourier 級数展開における関数の直交性) ex =
+(Fourier級数展開における関数の直交性) ex =
 
 
 
@@ -446,15 +446,15 @@ $$x\prime\cos x$$
 
 関数は無限個の直交関数列として表すことができる
 \= = a0 + a1 cos wx + a2 cos 2wx + · · ·
-例) フーリエ級数展開 y(x)
+例) フーリエ級数展開y(x)
 
 * b1 sin wx + b2 sin 2wx + · ·
-  関数の直交性 ← Euclid の基底ベクトルの直交性 (内積=0) の拡張
+  関数の直交性 ← Euclidの基底ベクトルの直交性 (内積=0) の拡張
 
 (0)
 
 
-三角関数の直交性 ex ・ey = ()
+三角関数の直交性ex ・ey = ()
 
 (1)
 
@@ -463,7 +463,7 @@ ex
 
 
 
-(Fourier 級数展開における関数の直交性)
+(Fourier級数展開における関数の直交性)
 rX
 X
 \~
@@ -511,7 +511,7 @@ y(x) = 12 a0 + a1 cos ωx + a2 cos 2ωx + ··· 例) フーリエ級数展開
 
 * b1 sin ωx + b2 sin 2ωx + ···
 
-a =   a1 a2 ...  関数 → 直交関数を基底とした無限次元空間内の一点  y(x) = !∞ i=1 aiφi(x) φi : i 番目の基底関数
+a =   a1 a2 ...  関数 → 直交関数を基底とした無限次元空間内の一点  y(x) = !∞ i=1 aiφi(x) φi : i番目の基底関数
 
 # 1. **連続体力学と変分法の関係**
 
@@ -522,8 +522,8 @@ $$\exists\exists\exists\forall i\exists i\exists i\exists i\exists i\exists i\ex
 関数は無限個の直交関数列として表すことができる
 
 y(x) = 12 a0 + a1 cos ωx + a2 cos 2ωx + ··· 例) フーリエ級数展開 + b1 sin ωx + b2 sin 2ωx + ···
-a =   a1 a2 ...  関数 → 直交関数を基底とした無限次元空間内の一点  y(x) = !∞ i=1 aiφi(x) φi : i 番目の基底関数
-偏微分方程式は関数を入力し、(初期条件等の) 関数を解の関数に移す写像 線形写像の場合は、無限次元空間内における線形代数学 関数解析学
+a =   a1 a2 ...  関数 → 直交関数を基底とした無限次元空間内の一点  y(x) = !∞ i=1 aiφi(x) φi : i番目の基底関数
+偏微分方程式は関数を入力し、(初期条件等の) 関数を解の関数に移す写像線形写像の場合は、無限次元空間内における線形代数学関数解析学
 
 
 # 1. **連続体力学と変分法の関係** 変分法における近似解法
@@ -532,14 +532,14 @@ a =   a1 a2 ...  関数 → 直交関数を基底とした無限次元空間内
 →**変分問題**
 (無限次元の微分: 汎関数微分)
 y!=
-dy 汎関数: 関数 → 数値 dx
+dy汎関数: 関数 → 数値dx
 
 解法: を満たす (停留関数) を求める
 →汎関数微分 (変分) **による解法**
 
 $$\cdot\prod^{\prime};,$$
 
-Euler–Lagrange 方程式 関数 → 直交関数を基底とした無限次元空間内の一点
+Euler–Lagrange方程式関数 → 直交関数を基底とした無限次元空間内の一点
 
 $$a=,$$
 
@@ -564,9 +564,9 @@ $$\exists\exists\exists\exists\exists\forall\exists\neg}$$
 →**変分問題**
 (無限次元の微分: 汎関数微分)
 y!=
-dy 汎関数: 関数 → 数値 dx 解法: を満たす (停留関数) を求める
+dy汎関数: 関数 → 数値dx解法: を満たす (停留関数) を求める
 →汎関数微分 (変分) **による解法**
-Euler–Lagrange 方程式 有限個の基底関数で近似
+Euler–Lagrange方程式有限個の基底関数で近似
 
 
 
@@ -595,10 +595,10 @@ i=1 aiφi(x)
 変分法 →汎関数の最大値・最小値問題
 →**変分問題**
 (無限次元の微分: 汎関数微分)
-汎関数: 関数 → 数値 y!=
-dy dx 解法: を満たす (停留関数) を求める
+汎関数: 関数 → 数値y!=
+dy dx解法: を満たす (停留関数) を求める
 →汎関数微分 (変分) **による解法**
-Euler–Lagrange 方程式 近似された関数に対する変分法 変分法の近似解法 (Ritz 法, Galerkin 法など)
+Euler–Lagrange方程式近似された関数に対する変分法変分法の近似解法 (Ritz法, Galerkin法など)
 
 
 有限個の基底関数で近似
@@ -606,7 +606,7 @@ Euler–Lagrange 方程式 近似された関数に対する変分法 変分法
 
 a =
 
-関数 → 直交関数を基底とした無限次元空間内の一点 a1 a2
+関数 → 直交関数を基底とした無限次元空間内の一点a1 a2
 ...
 
 y(x) = !∞
@@ -616,23 +616,23 @@ i=1 aiφi(x)
 
 # 目次
 
-1. 連続体力学と変分法の関係 2. **有限要素法の基本**
+1. 連続体力学と変分法の関係2. **有限要素法の基本**
    2.1. **重み付き残差法と弱形式化**
-   2.2. Galerkin 法 2.3. **形状関数による補間**
+   2.2. Galerkin法2.3. **形状関数による補間**
 
 2. **安定化有限要素法**
-   3.1. Upwind Galerkin 法 3.2. Streamline upwind Petrov/Gakerkin 法 3.3. **その他の安定化スキーム**
+   3.1. Upwind Galerkin法3.2. Streamline upwind Petrov/Gakerkin法3.3. **その他の安定化スキーム**
 
 3. **有限要素法の基本**
    有限要素法とは?
 
 
 偏微分方程式に対して重み付き残差法を適用し弱形式化した上で、
-メッシュの形状に対応した関数 (形状関数) を基底関数とした変分 法の近似解法 (Galerkin 法など) を用いて解析する手法 形状関数 **(shape function)**
+メッシュの形状に対応した関数 (形状関数) を基底関数とした変分法の近似解法 (Galerkin法など) を用いて解析する手法形状関数 **(shape function)**
 
 
 2.1. **重み付き残差法と弱形式化**
-2.2. Galerkin 法 2.3. 形状関数による補間
+2.2. Galerkin法2.3. 形状関数による補間
 
 ## Https://Www.Comsol.Jp/Multiphysics/Finite-Element-Method
 
@@ -641,8 +641,8 @@ i=1 aiφi(x)
 
 
 偏微分方程式に対して**重み付き残差法を適用し弱形式化**した上で、
-メッシュの形状に対応した関数 (形状関数) を基底関数とした変分 法の近似解法 (Galerkin 法など) を用いて解析する手法 2.1. **重み付き残差法と弱形式化**
-2.2. Galerkin 法 2.3. 形状関数による補間 形状関数 **(shape function)**
+メッシュの形状に対応した関数 (形状関数) を基底関数とした変分法の近似解法 (Galerkin法など) を用いて解析する手法2.1. **重み付き残差法と弱形式化**
+2.2. Galerkin法2.3. 形状関数による補間形状関数 **(shape function)**
 
 
 ## Https://Www.Comsol.Jp/Multiphysics/Finite-Element-Method
@@ -724,7 +724,7 @@ $$\partial\Omega=*{\star}\partial\Omega*{D}+\partial\Omega\_{\mu}$$
 $$\blacksquare$$
 
 Dirichlet **境界の表面**
-2.1. 重み付き残差法と弱形式化 重み付き残差法 **(Weighted Residual Method)**
+2.1. 重み付き残差法と弱形式化重み付き残差法 **(Weighted Residual Method)**
 重み付き残差方程式 **(weighted residual equation):**
 
 $$\int\_{\Omega}w(\mathbf{x})\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\cdot\nabla)u-\alpha\nabla^{2}u-f\right)d\Omega=0\qquad\mathbf{\cdot}\ \mathbf{\cdot}\ \mathbf{\cdot}\ \mathbf{(1)}$$
@@ -734,7 +734,7 @@ Neumann **境界の表面**
 Dirichlet **境界の表面**
 Dirichlet **境界条件**:
 u = uD
-解 u は境界条件を満たすように設定し、重み関数は境界上で 0 とする 2.1. 重み付き残差法と弱形式化 重み付き残差法 **(Weighted Residual Method)**
+解uは境界条件を満たすように設定し、重み関数は境界上で0とする2.1. 重み付き残差法と弱形式化重み付き残差法 **(Weighted Residual Method)**
 重み付き残差方程式 **(weighted residual equation):**
 
 $$\int\_{\Omega}w(\mathbf{x})\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\cdot\nabla)u-\alpha\nabla^{2}u-f\right)d\Omega=0\qquad\mathbf{\cdot}\ \mathbf{\cdot}\ \mathbf{\cdot}\ \mathbf{(1)}$$
@@ -744,7 +744,7 @@ Neumann **境界の表面**
 Dirichlet **境界の表面**
 Dirichlet **境界条件**:
 u = uD
-解 u は境界条件を満たすように設定し、重み関数は境界上で 0 とする Neumann **境界条件**:
+解uは境界条件を満たすように設定し、重み関数は境界上で0とするNeumann **境界条件**:
 α (∇u) · n = qN
 
 # 2.1. 重み付き残差法と弱形式化 重み付き残差法 **(Weighted Residual Method)**
@@ -758,7 +758,7 @@ Neumann **境界の表面**
 Dirichlet **境界の表面**
 Dirichlet **境界条件**:
 u = uD
-解 u は境界条件を満たすように設定し、重み関数は境界上で 0 とする Neumann **境界条件**:
+解uは境界条件を満たすように設定し、重み関数は境界上で0とするNeumann **境界条件**:
 α (∇u) · n = qN
 重み付き残差法を用いて定式化
 ∂ΩN
@@ -769,11 +769,11 @@ w(x) (α (∇u) · n − qN ) dΓ = 0
 
 $$\int\_{\Omega}w(\mathbf{x})\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\cdot\nabla)u-\alpha\nabla^{2}u-f\right)d\Omega=0\qquad\mathbf{\cdot,\cdot,,,(1)}$$
 
-Dirichlet **境界条件**: 解 u は境界条件を満たすように設定し、重み関数は境界上で 0 とする
+Dirichlet **境界条件**: 解uは境界条件を満たすように設定し、重み関数は境界上で0とする
 !
 
 ∂ΩN w(x) (α (∇u) · n − qN ) dΓ = 0 ・・・(2)
-Neumann 境界条件:
+Neumann境界条件:
 
 # 2.1. 重み付き残差法と弱形式化 重み付き残差法 **(Weighted Residual Method)**
 
@@ -782,7 +782,7 @@ $$\Xi\bigcup{}\_{\lambda}$$
 $$\frac{\pi}{2}$$  2.
 
 !
-Dirichlet **境界条件**: 解 u は境界条件を満たすように設定し、重み関数は境界上で 0 とする Neumann **境界条件**:
+Dirichlet **境界条件**: 解uは境界条件を満たすように設定し、重み関数は境界上で0とするNeumann **境界条件**:
 
 $$#\ u\ {\pm\frac{1}{2\pi}\mp\frac{1}{2\pi}\pm\frac{1}{1!!!!}}\pm\frac{1}{2\pi}\mp\frac{1}{2\pi}\pm\frac{1}{2\pi}$$
 $${\mathcal{H}}\_{\Gamma}^{\mu}:\qquad\oint$$
@@ -799,7 +799,7 @@ $$\left(2\right)$$
 $$\rho\psi(\infty)\left(\infty\left(\nabla\rho\right)\cdot\infty\right)$$
 
 ∂ΩN
-w(x) (α (∇u) · n − qN ) dΓ = 0 境界条件を考慮した重み付き残差方程式:
+w(x) (α (∇u) · n − qN ) dΓ = 0境界条件を考慮した重み付き残差方程式:
 
 $${\mathfrak{C}}\setminus{\mathfrak{C}}$$
 Ω
@@ -820,7 +820,7 @@ w(x)
 "∂u
 !
 ∂t
-Dirichlet 境界条件: 解 u は境界条件を満たすように設定し、重み関数は境界上で 0 とする ! ・・・(2) ∂ΩN w(x) (α (∇u) · n − qN ) dΓ = 0 Neumann 境界条件:
+Dirichlet境界条件: 解uは境界条件を満たすように設定し、重み関数は境界上で0とする ! ・・・(2) ∂ΩN w(x) (α (∇u) · n − qN ) dΓ = 0 Neumann境界条件:
 境界条件を考慮した重み付き残差方程式:
 
 $$\int\_{\Omega}w(\mathbf{x})\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\cdot\nabla)u-\alpha\nabla^{2}u-f\right)d\Omega+\int\_{\partial\Omega\_{N}}w(\mathbf{x})\left(\alpha\left(\nabla u\right)\cdot\mathbf{n}-q\_{N}\right)d\Gamma=0$$
@@ -829,8 +829,8 @@ $$\int\_{\Omega}w(\mathbf{x})\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\c
   \#dΩ −
   !
   ∂ΩN
-  w(x) (α (∇u) · n − qN ) dΓ = 0 重み関数 w は任意の値をとり、 上で 0 である場合も含む
-  →上式の右辺第2項が消え、Neumann 境界条件なしの式が成立
+  w(x) (α (∇u) · n − qN ) dΓ = 0重み関数wは任意の値をとり、 上で0である場合も含む
+  →上式の右辺第2項が消え、Neumann境界条件なしの式が成立
 
 # 2.1. 重み付き残差法と弱形式化 重み付き残差法 **(Weighted Residual Method)**
 
@@ -840,7 +840,7 @@ w(x)
 "∂u
 !
 ∂t
-Dirichlet 境界条件: 解 u は境界条件を満たすように設定し、重み関数は境界上で 0 とする
+Dirichlet境界条件: 解uは境界条件を満たすように設定し、重み関数は境界上で0とする
 Neumann **境界条件**:
 
 ! ・・・(2) ∂ΩN w(x) (α (∇u) · n − qN ) dΓ = 0
@@ -852,9 +852,9 @@ $$\int\_{\Omega}w(\mathbf{x})\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\c
   \#dΩ −
   !
   ∂ΩN
-  w(x) (α (∇u) · n − qN ) dΓ = 0 重み関数 w は任意の値をとり、 上で 0 である場合も含む
-  →上式の右辺第2項が消え、Neumann 境界条件なしの式が成立 u がもとの方程式を満たす (解である) 場合
-  →上式の右辺第1項が消え、Neumann 境界条件の式が成立
+  w(x) (α (∇u) · n − qN ) dΓ = 0重み関数wは任意の値をとり、 上で0である場合も含む
+  →上式の右辺第2項が消え、Neumann境界条件なしの式が成立uがもとの方程式を満たす (解である) 場合
+  →上式の右辺第1項が消え、Neumann境界条件の式が成立
 
 # 2.1. 重み付き残差法と弱形式化 弱形式化 **(Weak Formulation)**
 
@@ -862,8 +862,8 @@ $$\int\_{\Omega}w(\mathbf{x})\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\c
 
 $$\int\_{\Omega}w(x)\left({\frac{\partial u}{\partial t}}+(v\cdot\nabla)u-{\overline{{\left|\alpha\nabla^{2}u\right|}}}\right)$$
 $$\left({\frac{\partial u}{\partial t}}+\left(\mathbf{v}\cdot\nabla\right)u-{\boxed{\alpha\nabla^{2}u}}-f\right)d\Omega+\int\_{\partial\Omega\_{N}}w(\mathbf{x})\left(\alpha\left(\nabla u\right)\cdot\mathbf{n}-q\_{N}\right)d\Gamma=0$$
-※ 解 u は境界条件を満たすように設定し、重み関数は境界上で 0
-Gauss–Green の定理を使って、2階微分を1**階微分に変換** → 弱形式化 **(weak formulation)**
+※ 解uは境界条件を満たすように設定し、重み関数は境界上で0
+Gauss–Greenの定理を使って、2階微分を1**階微分に変換** → 弱形式化 **(weak formulation)**
 
 # 2.1. **重み付き残差法と弱形式化** 弱形式化 **(Weak Formulation)**
 
@@ -873,7 +873,7 @@ $$\int\_{\Omega}w(\mathbf{x})\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\c
 $$\begin{array}{c}{{\left({\frac{\partial u}{\partial t}}+\left(\mathbf{v}\cdot\nabla\right)u-\left|\alpha\nabla^{2}u\right|-f\right)d\Omega+\int\_{\partial\Omega\_{N}}w(\mathbf{x})\left(\alpha\left(\nabla u\right)\cdot\mathbf{n}-q\_{N}\right)d\Gamma=0}}\ {{\left.\right|}}\ {{\left.\right|}}\ {{\left.\right|}}\end{array}$$
 Ω
 α(∇w) · (∇u)dΩ
-Gauss–Green の定理を使って、2階微分を1階微分に変換
+Gauss–Greenの定理を使って、2階微分を1階微分に変換
 → 弱形式化 **(weak formulation)**
 
 # $$\int\_{\Omega}w(\alpha\nabla^{2}u)d\Omega=\int\_{\partial\Omega}w\alpha(\nabla u)\cdot n d\Gamma-\int\_{\Omega}\alpha(\nabla w)\cdot(\nabla u)d\Omega$$
@@ -887,14 +887,14 @@ wα(∇u) · ndΓ −
 
 境界条件を考慮した重み付き残差方程式:
 
-! Ω w(x) "∂u ∂t + (v · ∇)u − α∇2u − f #dΩ + ! ∂ΩN w(x) (α (∇u) · n − qN ) dΓ = 0 ※ 解 u は境界条件を満たすように設定し、重み関数は境界上で 0 Gauss–Green の定理を使って、2階微分を1階微分に変換 → 弱形式化 (weak formulation) ! Ω w(α∇2u)dΩ = ! ∂Ω wα(∇u) · ndΓ − ! ! Ω w(α∇2u)dΩ = ! ∂Ω wα(∇u) · ndΓ − ! Ω α(∇w) · (∇u)dΩ Ω α(∇w) · (∇u)dΩ = ! ∂ΩN wα(∇u) · ndΓ − ! = ! ∂ΩN wα(∇u) · ndΓ − ! Ω α(∇w) · (∇u)dΩ Ω α(∇w) · (∇u)dΩ
+! Ω w(x) "∂u ∂t + (v · ∇)u − α∇2u − f #dΩ + ! ∂ΩN w(x) (α (∇u) · n − qN ) dΓ = 0 ※ 解uは境界条件を満たすように設定し、重み関数は境界上で0 Gauss–Greenの定理を使って、2階微分を1階微分に変換 → 弱形式化 (weak formulation) ! Ω w(α∇2u)dΩ = ! ∂Ω wα(∇u) · ndΓ − ! ! Ω w(α∇2u)dΩ = ! ∂Ω wα(∇u) · ndΓ − ! Ω α(∇w) · (∇u)dΩ Ω α(∇w) · (∇u)dΩ = ! ∂ΩN wα(∇u) · ndΓ − ! = ! ∂ΩN wα(∇u) · ndΓ − ! Ω α(∇w) · (∇u)dΩ Ω α(∇w) · (∇u)dΩ
 ∂Ω = ∂ΩD + ∂ΩN
 Neumann **境界の表面**
 Dirichlet **境界の表面**
 
 # 2.1. **重み付き残差法と弱形式化** 弱形式化 **(Weak Formulation)**
 
-! Ω w(x) "∂u ∂t + (v · ∇)u − α∇2u − f #dΩ + ! ∂ΩN w(x) (α (∇u) · n − qN ) dΓ = 0 ※ 解 u は境界条件を満たすように設定し、重み関数は境界上で 0 Gauss–Green の定理を使って、2階微分を1階微分に変換 → 弱形式化 (weak formulation)
+! Ω w(x) "∂u ∂t + (v · ∇)u − α∇2u − f #dΩ + ! ∂ΩN w(x) (α (∇u) · n − qN ) dΓ = 0 ※ 解uは境界条件を満たすように設定し、重み関数は境界上で0 Gauss–Greenの定理を使って、2階微分を1階微分に変換 → 弱形式化 (weak formulation)
 境界条件を考慮した重み付き残差方程式:
 
 
@@ -907,10 +907,10 @@ $$\frac{\operatorname\*{lim}*{(\nabla u)\cdot\mathcal{M}}}{\operatorname\*{lim}*
 !
 $$\begin{array}{c}{{r\atop\Omega}}\ {{w(\mathbf{x})\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\cdot\nabla)u-\overline{{{\left|\alpha\nabla^{2}u\right|}}}-f\right)d\Omega+\int\_{\partial\Omega\_{N}}w(\mathbf{x})\left(\underline{{{\alpha\mathbf{\Sigma^{*}\mathbf{\Sigma^{*}\mathbf{n}}}}}}-q\_{N}\right)d\Gamma=0}}\end{array}$$
 !
-※ 解 u は境界条件を満たすように設定し、重み関数は境界上で 0
+※ 解uは境界条件を満たすように設定し、重み関数は境界上で0
 
 
-Gauss–Green の定理を使って、2階微分を1**階微分に変換**
+Gauss–Greenの定理を使って、2階微分を1**階微分に変換**
 
 
 
@@ -932,18 +932,18 @@ $$\prod\_{\Omega},\alpha(\nabla w)\cdot(\nabla u)d\Omega$$
 Ω
 w
 $$\left({\frac{\partial u}{\partial t}}+(\mathbf{v}\cdot\nabla)u-f\right)d\Omega+\int\_{\Omega}\alpha\left(\nabla w\right)\cdot\left(\nabla u\right)d\Omega-\int\_{\partial\Omega\_{N}}w q\_{N}d\Gamma=0$$
-※ 解 u は Dirichlet 境界条件を満たすように設定
+※ 解uはDirichlet境界条件を満たすように設定
 
 2. **有限要素法の基本**
    有限要素法とは?
 
 
 偏微分方程式に対して重み付き残差法を適用し弱形式化した上で、
-メッシュの形状に対応した関数 (形状関数) を基底関数とした変分 法の近似解法 (Galerkin 法など) を用いて解析する手法 形状関数 **(shape function)**
+メッシュの形状に対応した関数 (形状関数) を基底関数とした変分法の近似解法 (Galerkin法など) を用いて解析する手法形状関数 **(shape function)**
 
 
 2.1. **重み付き残差法と弱形式化**
-2.2. Galerkin 法 2.3. 形状関数による補間
+2.2. Galerkin法2.3. 形状関数による補間
 
 ## Https://Www.Comsol.Jp/Multiphysics/Finite-Element-Method
 
@@ -964,8 +964,8 @@ w
 
 ∂ΩN
 wqN dΓ = 0
-※ 解 u は Dirichlet 境界条件を満たすように設定 Galerkin 法:
-重み関数と解を有限個の基底関数で近似した上で、双方に同一の基底 関数を用いる手法
+※ 解uはDirichlet境界条件を満たすように設定Galerkin法:
+重み関数と解を有限個の基底関数で近似した上で、双方に同一の基底関数を用いる手法
 
 
 $\mathbf{a}\in\mathbb{R}^n$
@@ -989,11 +989,11 @@ $\square$
 ∂ΩN
 wqN dΓ = 0
 
-※ 解 u は Dirichlet 境界条件を満たすように設定 Galerkin 法:
-重み関数と解を有限個の基底関数で近似した上で、双方に同一の基底 関数を用いる手法
+※ 解uはDirichlet境界条件を満たすように設定Galerkin法:
+重み関数と解を有限個の基底関数で近似した上で、双方に同一の基底関数を用いる手法
 
 $$w(x)\simeq w\_{h}(x)=\sum\_{i=1}^{m}w\_{i}N\_{i}(x)$$
 $$M$$
 $${\dot{\imath}}{\underline{{=}}}1$$
-u(x) ! uh(x) = ! i=1 uiNi(x) ※ 解 u は Dirichlet 境界条件を満たすように設定
+u(x) ! uh(x) = ! i=1 uiNi(x) ※ 解uはDirichlet境界条件を満たすように設定
 $$M$$
