diff --git a/tmp/tmp.IxkReqOrfD b/tmp/tmp.fltFim151y
index 3ab3ea2..4782389 100644
--- a/tmp/tmp.IxkReqOrfD
+++ b/tmp/tmp.fltFim151y
@@ -1,6 +1,7 @@
-重も自分子ード分解法は、
-ラえられたデータの動的な 情報や構造を捉えること を目的として開発された キ法である。
+## Markdown Content
 
- 数値解析の時間ステップに に対する計算語果と2k とする.
+重み付き自己コード分解法は、与えられたデータの動的な情報や構造を捉えることを目的として開発された手法である。
 
-といえ、具体的に17. 数値解析 て用いる 6 ) 編 相みたんでる
\ No newline at end of file
+数値解析の時間ステップに対する計算結果を $2k$ とする。
+
+ここで、具体的に数値解析で用いるには、さらに検討が必要である。
\ No newline at end of file
